sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -258,6 +258,8 @@ public TaskManagementSteps(
     When(
         "I check if popup message is {string}",
         (String expectedText) -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(
+              By.cssSelector(".v-Notification-description"));
```

Comment:
create locator in page object for notification popup

Reply:
done